src/engine/InputQuery.cpp

```diff
@@ -362,14 +362,18 @@ void InputQuery::saveQuery( const String &fileName )
     // Number of Equations
     queryFile->write( Stringf( "%u\n", _equations.size() ) );
 
-    // Number of Constraints
-    queryFile->write( Stringf( "%u", _plConstraints.size() ) );
+    // Number of Piecewise-Liner Constraints
+    queryFile->write( Stringf( "%u\n", _plConstraints.size() ) );
+
+    // Number of Transcendental Constraints
+    queryFile->write( Stringf( "%u", _tsConstraints.size() ) );
 
     printf( "Number of variables: %u\n", _numberOfVariables );
     printf( "Number of lower bounds: %u\n", _lowerBounds.size() );
     printf( "Number of upper bounds: %u\n", _upperBounds.size() );
     printf( "Number of equations: %u\n", _equations.size() );
-    printf( "Number of constraints: %u\n", _plConstraints.size() );
+    printf( "Number of piecewise-linear constraints: %u\n", _plConstraints.size() );
+    printf( "Number of transcendental constraints: %u\n", _tsConstraints.size() );
```

Comment:
Maybe we can just replace _plConstraints with _nonLinearConstaints which include both the PL constraints and the transcendental constraints. This way the previously generated input query file would remain valid.